.remarkrc.mjs

```diff
@@ -14,9 +14,9 @@ const remarkConfig = {
 			{
 				schemas: {
 					/* One schema for many files */
-					"./.github/linters/metadata.schema.yml": [
+					'./.github/linters/metadata.schema.yml': [
```

Comment:
Was this change made by IDE or on purpose?

Reply:
Not by IDE, but not necessarily on purpose. I was looking at the config example in the [plugin docs](https://github.com/JulianCataldo/remark-lint-frontmatter-schema#globally-with-patterns) and used single quotes to match. I'm happy to revert to keep consistent with other repos though.

Reply:
Never mind, I checked the file and it was already inconsistent.